plugin/src/test/resources/com/h3xstream/findsecbugs/xss/CustomConfig.txt

```diff
@@ -3,4 +3,8 @@ javax/servlet/http/HttpServletRequest.getAttribute("tainted"):TAINTED@org/apache
 
 javax/servlet/http/HttpSession.getAttribute(UNKNOWN):SAFE@org/apache/jsp/xss/xss_005f8_005frequest_005fattribute_jsp
 
-org/apache/jsp/xss/xss_005f8_005frequest_005fattribute_jsp.suffix(TAINTED,SAFE,"_suffix"):SAFE@org/apache/jsp/xss/xss_005f8_005frequest_005fattribute_jsp
\ No newline at end of file
+org/apache/jsp/xss/xss_005f8_005frequest_005fattribute_jsp.suffix(TAINTED,SAFE,"_suffix"):SAFE@org/apache/jsp/xss/xss_005f8_005frequest_005fattribute_jsp
```

Comment:
@h3xstream yes, I would agree.  I did not add this line (probably shows due to whitespace differences), but believe it could be changed to remove the filename with the introduction of TaintMethodConfigurationWithArguments.

Reply:
I still don't understand what new features this PR is bringing. 

I think I am missing something .. I only see one test case where the API `req.getAttribute()` is considered safe.

Reply:
@h3xstream  This is made more confusing by to my recent lack of time to complete this. My original intent was to remove FPs (found in 1.6) from the OWASP taglib, but have been unable to get a failure in the newer versions of FSB.  It's possible the FPs have been removed via some other means, but I haven't had time to validate, or to otherwise get a good starting point (i.e presence of the FP in the FSB bug report) in order to add Custom Configuration.
This PR does contribute a generalization of TaintMethodConfigWithArgumentsAndLocation, called TaintMethodConfigWithArguments,  which enables TaintMethodConfiguration without a location, and therefore allowed [this](https://github.com/bradflood/find-sec-bugs/blob/a6b37720d17cf027ae1e13e30de731a8b38b512e/plugin/src/test/resources/com/h3xstream/findsecbugs/xss/CustomConfig.txt#L10) configuration, which removes FPs when the JSP only refers to container-controlled objects ([example](https://github.com/bradflood/find-sec-bugs/blob/a6b37720d17cf027ae1e13e30de731a8b38b512e/plugin/src/test/webapp/xss/xss_9_jsp_container.jsp#L1))

I did not modify CustomConfig.txt line 6 to reflect the new ability to remove location from the custom taint configuration because I did not write that initially. I will complete this. 

Also, this PR fixes a small bug in [BugInstanceMatcher](https://github.com/bradflood/find-sec-bugs/blob/a6b37720d17cf027ae1e13e30de731a8b38b512e/findbugs-test-util/src/test/java/com/h3xstream/findbugs/test/matcher/BugInstanceMatcher.java#L131) in which replaceAll was too aggressive with JSP files